plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt

```diff
@@ -467,37 +567,55 @@ open class DefaultPageCreator(
             }
         }
 
-    private fun DocumentableContentBuilder.functionsBlock(name: String, list: Collection<DFunction>) = divergentBlock(
+    private fun DocumentableContentBuilder.functionsBlock(
+        name: String,
+        list: Collection<DFunction>,
+        isVisibleHeader: Boolean = false
+    ) = divergentBlock(
         name,
         list.sorted(),
         ContentKind.Functions,
-        extra = mainExtra + SimpleAttr.header(name)
+        extra = mainExtra + SimpleAttr.togglableTarget(name),
+        headExtra = if(isVisibleHeader) mainExtra + SimpleAttr.togglableTarget(name) else mainExtra
     )
 
     private fun DocumentableContentBuilder.propertiesBlock(
         name: String,
         list: Collection<DProperty>,
-        sourceSets: Set<DokkaSourceSet>
+        sourceSets: Set<DokkaSourceSet>,
+        isVisibleHeader: Boolean = false
     ) {
+        val groupedElements = list.groupBy { Pair(it.name, it.isExtension()) }.toList()
+        val sortedGroupedElements = groupedElements.sortedWith(compareBy<Pair<Pair<String, Boolean>, List<DProperty>>, String>(nullsLast(String.CASE_INSENSITIVE_ORDER)) { it.first.first }.thenBy { it.first.second })
+
```

Comment:
I wonder if the implementation of `propertiesBlock` could be switched to `divergentBlock` like for `functionsBlock`. Properties and functions should be pretty close in handling - both have extensions, source sets, need to be sorted, etc. Unless I'm missing something

I tried it locally and it seems to work

Reply:
`divergentBlock` has `after` and `before` to group overloads.

Reply:
Does it cause problems? As far as I can see, properties with the same name are grouped just like functions. The only difference now is that the duplicate KDocs are rendered separately for properties, whereas they're the same when it comes to functions

<details>
<summary>Example</summary>

```kotlin
class MyClass {

    /**
     * One
     */
    val String.overload: String
        get() = "asd"

    /**
     * DUPLICATE
     */
    val Int.overload: Int
        get() = 42

    /**
     * DUPLICATE
     */
    val Long.overload: Long
        get() = 42L


    /**
     * DUPLICATE
     */
    fun foo() {

    }

    /**
     * DUPLICATE
     */
    fun foo(s: String) {

    }
}
```

![2023-02-09_00-10-18](https://user-images.githubusercontent.com/22685910/217672425-d9aa4710-77f8-4a3b-91f4-0cca268ee568.png)

![2023-02-09_00-10-13](https://user-images.githubusercontent.com/22685910/217672429-4269a61d-84ca-49cb-baf7-d91468534e13.png)

</details>

Reply:
oops, my comment is cut(
 this grouping  has been here before my PR. I have changed nothing. I think It deserves another PR.